includes/commerce_pos.transaction.inc

```diff
@@ -183,23 +183,24 @@ function commerce_pos_transaction_form(&$form, &$form_state, $transaction_type)
       '#tree' => TRUE,
       '#suffix' => '</div>',
     );
+    
     $form['order_search']['input'] = array(
       '#type' => 'textfield',
       '#title' => '',
       '#size' => 60,
-      '#description' => t('Search by order ID to load the items in an order.<br>(Note: This does not support searching for Order Number.)'),
+      '#description' => t('Search by order number to load the items in an order.'),
       '#attributes' => array(
         'placeholder' => t('Order Search'),
       ),
+      '#element_validate' => array('element_validate_integer_positive'),
       '#default_value' => !empty($form_state['values']['order_search']['input']) ? $form_state['values']['order_search']['input'] : NULL,
     );
 
     $form['order_search']['search'] = array(
       '#type' => 'button',
-      '#value' => t('Go'),
+      '#value' => t('Return items from this order'),
```

Comment:
This seems a little long to me... Maybe something like "Lookup Order" would be more succinct?